test/k8sT/Services.go

```diff
@@ -412,6 +412,141 @@ var _ = Describe("K8sServicesTest", func() {
 		})
 	})
 
+	SkipContextIf(func() bool { return !helpers.RunsOnNetNextOr419Kernel() }, "Checks local redirect policy", func() {
+		const (
+			lrpServiceName = "lrp-demo-service"
+			fe1Name        = "k8s1-client"
+			fe2Name        = "k8s2-client"
+			be1Name        = "k8s1-backend"
+			be2Name        = "k8s2-backend"
+			repeat         = 10
+			feFilter       = "role=frontend"
+			beFilter       = "role=backend"
+		)
+
+		var (
+			deploymentYAML string
+			ctx            context.Context
+			ctxCancel      context.CancelFunc
+			svcIP          string
+			curlTCP        string
+			curlUDP        string
+		)
+
+		BeforeAll(func() {
+			deploymentYAML = helpers.ManifestGet(kubectl.BasePath(), "lrp-test.yaml")
+			res := kubectl.ApplyDefault(deploymentYAML)
+			res.ExpectSuccess("Unable to apply %s", deploymentYAML)
+			pods := []string{feFilter, beFilter}
+			for _, pod := range pods {
+				err := kubectl.WaitforPods(helpers.DefaultNamespace, fmt.Sprintf("-l %s", pod), helpers.HelperTimeout)
+				ExpectWithOffset(1, err).Should(BeNil())
```

Comment:
I don't think this needs to be an `ExpectWithOffset`. That's reserved for when you have a helper function that's called from inside an `It()` block which performs an assertion. Hence, needing an offset because it's inside a function, rather than in the `It()` itself. This is relevant when the test fails and you get a stacktrace of where the failure occurred.

Reply:
Done.